ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterManager.java

```diff
@@ -677,5 +701,20 @@ long getAllocatedUsableCapacity() {
       return clusterWideAllocatedUsableCapacityBytes;
     }
   }
+
+  /**
+   * @return the current xid;
+   */
+  protected long getCurrentXid() {
```

Comment:
This method is for future use, right? (looks like is never invoked in this PR)

Reply:
This is a getter and setter for the current xid config of the Helix cluster manager (not the InstanceConfigs), so the bootstrap tool does not affect this. This is primarily for helping test the code, but in the future, if there is a way to dynamically set the configs then these can potentially be useful.

The other option is to reinitialize the `HelixClusterManager`.

Reply:
Removed these methods. Testing those flows by simulating the advancement of the current xid by moving back the xid in the `InstanceConfig`s.